src/Microsoft.Framework.Runtime.Hosting/ProjectFilesCollection.cs

```diff
@@ -13,7 +13,7 @@ public class ProjectFilesCollection
         public static readonly string[] DefaultBundleExcludePatterns = new[] { @"obj/**/*.*", @"bin/**/*.*", @"**/.*/**" };
         public static readonly string[] DefaultPreprocessPatterns = new[] { @"compiler/preprocess/**/*.cs" };
         public static readonly string[] DefaultSharedPatterns = new[] { @"compiler/shared/**/*.cs" };
-        public static readonly string[] DefaultResourcesPatterns = new[] { @"compiler/resources/**/*" };
+        public static readonly string[] DefaultResourcesPatterns = new[] { @"compiler/resources/**/*.resx" };
```

Comment:
This isn't right. There should be 2 default resource patterns. This array should have:

```
public static readonly string[] DefaultResourcesPatterns = new[] { @"compiler/resources/**/*", "**/*.resx" };
```